### PR TITLE
Installation Tests

### PR DESCRIPTION
**Pull request by SolAZDev**
## Problem

The config and `pushd` directories are wrong. `$final_path` is also unbound.

## Solution

Fixed a typo ("etc/" -> "/etc/") and hardcoded install directory ($final_path -> /var/www/motioneye)

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme :facepalm:

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -4,23 +4,21 @@ packaging_format = 2
 
 id = "motioneye"
 name = "motionEye"
-description.en = "Explain in *a few (10~15) words* the purpose of the app or what it actually does (it is meant to give a rough idea to users browsing a catalog of 100+ apps)"
-description.fr = "Expliquez en *quelques* (10~15) mots l'utilité de l'app ou ce qu'elle fait (l'objectif est de donner une idée grossière pour des utilisateurs qui naviguent dans un catalogue de 100+ apps)"
+description.en = "motionEye is a web frontend for the motion daemon, written in Python."
+description.fr = "motionEye is a web frontend for the motion daemon, written in Python"
 
-version = "1.0~ynh1"
+version = "ae136a4c45034839db10388dd7cd5ad7ecfe2f21.2023.07.31"
```
Comment (Tagadda) [inline manifest.toml:10]: I guess the version number is wrong? I would use this:
```suggestion
version = "0.2023.07.31"
```

File: manifest.toml [hunk 2]
```diff
@@ -4,23 +4,21 @@ packaging_format = 2
...
 license = "free"
 website = "https://github.com/motioneye-project/motioneye"
-demo = "https://demo.example.com"
-admindoc = "https://yunohost.org/packaging_apps"
-userdoc = "https://yunohost.org/apps"
+userdoc = "https://github.com/motioneye-project/motioneye/wiki"
 code = "https://github.com/motioneye-project/motioneye"
 # FIXME: optional but recommended if relevant, this is meant to contain the Common Platform Enumeration, which is sort of a standard id for applications defined by the NIST. In particular, YunoHost may use this is in the future to easily track CVE (=security reports) related to apps. The CPE may be obtained by searching here: https://nvd.nist.gov/products/cpe/search. For example, for Nextcloud, the CPE is 'cpe:2.3:a:nextcloud:nextcloud' (no need to include the version number)
-cpe = "???"
+cpe = "https://nvd.nist.gov/products/cpe/detail/7E5CD831-B595-4643-9346-B11D2D1FEBC7?namingFormat=2.3&orderBy=CPEURI&keyword=motioneye&status=FINAL"
```
Comment (Tagadda) [inline manifest.toml:21]:
```suggestion
cpe = "cpe:2.3:a:motioneye_project:motioneye"
```

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

File: manifest.toml [hunk 3]
```diff
@@ -51,7 +51,7 @@ ram.runtime = "1024M"
 
         [resources.sources.main]
         url = "https://github.com/motioneye-project/motioneye/archive/refs/tags/0.43.1b1.tar.gz"
```
Comment (OniriCorpe) [inline manifest.toml:53]: usually, Yunohost packages stay with stable versions and avoid pre-releases

File: conf/motioneye.conf [hunk 4]
```diff
@@ -1,14 +1,14 @@
 # path to the configuration directory (must be writable by motionEye)
-conf_path /etc/motioneye
+conf_path /var/www/__APP__
 
 # path to the directory where pid files go (must be writable by motionEye)
 run_path /run/motioneye
 
 # path to the directory where log files go (must be writable by motionEye)
-log_path /var/log/motioneye
+log_path __DATA_DIR__/logs
```
Comment (OniriCorpe) [inline conf/motioneye.conf:8]: why move logs on a non standard path?

File: manifest.toml [hunk 5]
```diff
@@ -4,53 +4,37 @@ packaging_format = 2
...
-# FIXME: replace with true, false, or "not_relevant". Not to confuse with the "ldap" key: the "sso" key corresponds to wether or not a user is *automatically logged-in* on the app when logged-in on the YunoHost portal.
-sso = "?"
-# FIXME: replace with an **estimate** minimum disk and RAM requirements. e.g. 20M, 400M, 1G...
-disk = "50M"
-ram.build = "50M"
-ram.runtime = "50M"
+ldap = false
+sso = true
+disk = "500M"
+ram.build = "5120M"
```
Comment (OniriCorpe) [inline manifest.toml:26]: do you really mean 5 GB of RAM are needed to install it?

Comment (SolAZDev) [reply manifest.toml:26]: thanks for pointing it out I meant it to have 512M

**Comment by OniriCorpe** [comment]
https://github.com/YunoHost-Apps/motioneye_ynh/pull/11/commits/bd1c74561f321aca987d9ee18808603a0287d387

the standard for log folder is /var/log/$app ^w^

File: manifest.toml (hunk 3 above)
Comment (SolAZDev) [reply manifest.toml:53]: The stable release in this ocassion is a python2 base build, while this beta release is the python3 rewrite

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
!testme

**Comment by SolAZDev** [comment]
Install tests work fine, so I'll merge this. 
This is now using python3 and the 0.43.1b1 beta pre-release. 
As of now it's considered in-progress, due to the fact that it is a beta version.

**Pull request merged**